LICENSE

```diff
@@ -0,0 +1,27 @@
+Unless it has its own copyright/license embedded in its body, each source file
```

Comment:
`s/source//` to cater for documentation files etc.

But, there is a more fundamental problem with this statement: some files in `libyaml/` do not have any copyright/license statement and they are neither copyrighted by Linaro nor covered by BSD 2-Clause...

Reply:
Do we copyright the docs? especially since docs have moved to `optee_docs`?
Yet, ok to remove reference to "source". Maybe this change could also replace the license notice from source and header files with a SPDX license tag.

Regarding yaml, libyaml is release under MIT license terms. Maybe both license terms (MIT and BSD-2-Clause)  should be listed in this `LICENSE` file.